migrations/2023-04-06-092008_create_merchant_ek/up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE merchant_key_store(
+    merchant_id VARCHAR(255) NOT NULL PRIMARY KEY,
+    key bytea NOT NULL,
+    created_at TIMESTAMP NOT NULL DEFAULT now()
+);
+
+CREATE UNIQUE INDEX merchant_key_store_unique_index ON merchantkeystore(merchant_id);
```

Comment:
I'm pretty sure this query is not required since `PRIMARY KEY` implies a unique index. If it is required, update the table name to the correct one.